### PR TITLE
[WB-8550] Upgrade to click > 8.0.0

### PR DESCRIPTION
**Pull request by dmitryduev**
Fixes WB-8550

Description
-----------
Upgrade to `click > 8.0.0`.

Testing
-------
How was this PR tested?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Click>=7.0,!=8.0.0
+Click>8.0.0
```
Comment (kptkin) [inline requirements.txt:1]: should we just try?
```suggestion
Click!=8.0.0
```

Comment (dmitryduev) [reply requirements.txt:1]: With this PR, I just wanted to check if anything breaks if we require>8.0.0. Nothing seems to break so I'd close the issue. 
We do need the lower bound though, see https://click.palletsprojects.com/en/8.0.x/upgrading/#upgrading-to-7-0 (<7.0 would likely need special treatment).

Comment (kptkin) [reply requirements.txt]: ah got it about >7.0 what about `==8.0.0`?

Comment (kptkin) [reply requirements.txt]: ah you are already did it... funny

**Comment by dmitryduev** [comment]
We require `Click>=7.0,!=8.0.0`, (see [client/requirements.txt at master · wandb/client](https://github.com/wandb/client/blob/master/requirements.txt#L1) ) so it’s only about the specific version that was broken (`8.0.0`; the issue that was causing problems was fixed in `8.0.1`, see [flag_value converted to str in Click 8 · Issue #1886 · pallets/click](https://github.com/pallets/click/issues/1886) ). Any other `8.0.x` version will work fine with `wandb`.
Closing.

File: requirements.txt (hunk 1 above)
Comment (dmitryduev) [reply requirements.txt]: @dannygoldstein figured this out a long time ago :)
https://github.com/wandb/client/pull/2261

Comment (kptkin) [reply requirements.txt]: ah should we leave a comment for future investigations?

**Pull request closed without merging**